src/test/bbb.rs

```diff
@@ -218,6 +218,7 @@ fn bbb() {
                     ..Default::default()
                 },
                 mdia: Mdia {
+                        unexpected: vec![],
```

Comment:
Different indent, and in at least one other place. Probably just copied in. Maybe `cargo fmt`?